lib/line-protocol.js

```diff
@@ -169,13 +169,17 @@ module.exports.format = (event, config) => {
         return;
     }
 
-    const tags = internals.serialize(internals.tags(event));
-    const eventValues = getEventValues(event);
+    // Tag set
+    const tagsObject = internals.tags(event);
     if (config.metadata) {
```

Comment:
I would put this logic right in the `internals.tags` function.  Seems like a cleaner implementation to me.